apps/gitness/src/pages-v2/repo/repo-code.tsx

```diff
@@ -0,0 +1,148 @@
+import { useEffect, useMemo, useState } from 'react'
+import { useParams } from 'react-router-dom'
+
+import {
+  GitPathDetails,
+  OpenapiContentInfo,
+  OpenapiGetContentOutput,
+  pathDetails,
+  useFindRepositoryQuery,
+  useGetContentQuery
+} from '@harnessio/code-service-client'
+import { RepoFile, RepoFiles, SummaryItemType } from '@harnessio/ui/views'
+
+import { useGetRepoRef } from '../../framework/hooks/useGetRepoPath'
+import { timeAgoFromISOTime } from '../../pages/pipeline-edit/utils/time-utils'
+import { PathParams } from '../../RouteDefinitions'
+import { getTrimmedSha, normalizeGitRef } from '../../utils/git-utils'
+import { splitPathWithParents } from '../../utils/path-utils'
+
+/**
+ * TODO: This code was migrated from V2.
+ * @constructor
+ */
```

Comment:
Again, this isn't a class, so `@constructor` doesn't apply and the TODO isn't really a thing to do 🤔 
```suggestion
```